.gitignore

```diff
@@ -9,3 +9,4 @@
 .eslintrc.js
 .pa11yci.js
 .scss-lint.yml
+*.env*
```

Comment:
I know that this component doesn't need a `.env` file. But this is good practise nevertheless